### ISSUE TITLE
Add caption and alt text to image component

### ISSUE THREAD

**Issue by TimonVS** [storyblok/storyblok#285]
**The feature would affect:** (check one with "x")   
- [x] *app.storyblok.com (CMS - Interface)*   
- [x] *api.storyblok.com (CMS - Content Delivery API)*
- [ ] *mapi.storyblok.com (CMS - Management API)*
- [ ] *capi.storyblok.com (Commerce - API)*     
- [ ] *Commerce - Interface*   
- [ ] *Other* 

**Is your feature request related to a problem? Please describe.**

Alt text for images is tremendously important for websites. Currently to add alt text you have to define a separate field. The problem is that we'll have to define this everywhere the image component is being used because there's no way to share components outside of blocks.

This is how it's done in WordPress:

<img width="1525" alt="image" src="https://user-images.githubusercontent.com/876666/81823518-ae1a0400-9534-11ea-98ce-f5e81c2f36da.png">

**Describe the solution you'd like**

I'd like additional fields for caption and alt text for the image component field. They may be hidden away in an "advanced settings" modal, but I think alt text should be front and center because accessibility is important.

**Describe alternatives you've considered**

A custom component is not possible because they can only be used in blocks.

**Additional context**

**Comment by DominikAngerer** [comment]
Already planned for Q2: https://www.storyblok.com/features#roadmap
